script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -63,13 +63,21 @@ class PublishPluginCommand extends PluginCommand {
       // Flutter convention is to use "upstream" for the single source of truth, and "origin" for personal forks.
       defaultsTo: 'upstream',
     );
+    argParser.addOption(
+      _allOption,
+      help:
+          'Based the diff between $_kBaseShaOption and HEAD, publish all the plugins that has pubspec change in this diff.\n'
```

Comment:
Do we need to handle the case where there have been multiple releases of a single plugin in the range, or are we going to avoid that issue in automation by never doing a range spanning multiple changes?

Reply:
The issue won't exist when we start the automate process, because every commit that contains a pubspec change is going to run a publish command. 

There might be leftovers before the automation so we might need to handle that, handle it manually before automation doesn't seem to be too bad.